pubsub/server_test.go

```diff
@@ -254,24 +296,38 @@ func TestServerPublishSpecific(t *testing.T) {
 	}()
 	// not receive from the other
 	go func() {
-		err := webCon2.SetReadDeadline(time.Now().Add(5 * time.Second))
+		defer clientSideClosedConnection.Done()
+		err := webCon2.SetReadDeadline(time.Now().Add(50 * time.Millisecond))
 		require.NoError(err, "Error setting connection deadline.")
 		// Make sure connection wasn't written too
 		_, _, err = webCon2.ReadMessage()
 		require.Error(err, "Error not thrown.") //nolint:forbidigo
 		netErr, ok := err.(net.Error)
-		require.True(ok, "Error is not a net.Error")
+		require.True(ok, "Error is not a net.Error ( %v )", netErr)
```

Comment:
nit: should this be `True` with net error as a separate argument rather than including the formatter in the msg ?